Rubberduck.Core/VBERuntime/VBESettings.cs

```diff
@@ -24,7 +24,7 @@ public VBESettings(IVBE vbe, IRegistryWrapper registry)
         {
             try
             {
-                switch (Convert.ToInt32(decimal.Parse(vbe.Version)))
+                switch (int.Parse(vbe.Version.Split('.')[0]))
```

Comment:
oooh, good catch! 👍  

Memo to myself: think internationally -- they don't use `.` as the separator everywhere...